src/js/providers/html5.js

```diff
@@ -419,9 +420,13 @@ define([
                 return;
             }
             _videotag.removeAttribute('src');
-            if (!_isIE) {
+            if (!_isMSIE) {
                 _videotag.load();
             }
+            // IE may continue to play a video after changing source and loading a new media file.  JW7-1705
+            if(utils.isIETrident()) {
```

Comment:
@jw-kaurand @donato logged the bug with M$ https://connect.microsoft.com/IE/feedbackdetail/view/2000141/htmlmediaelement-autoplays-after-src-is-changed-and-load-is-called